### DISCUSSION TITLE
React App Github pages issue

### DISCUSSION THREAD

**Discussion by michaelanokyej** [community/community#23303] [Other Feature Feedback, Questions, & Ideas]
<p>Github pages is showing my README and not my actual website. My repo is <a href="https://github.com/michaelanokyej/portfolio-manager" rel="nofollow noopener">https://github.com/michaelanokyej/portfolio-manager</a>. Can someone please help me with this issue?</p>

**Comment by darrowby385** [comment, marked as answer]
<p>Hi, <a class="mention" href="/michaelanokyej">@michaelanokyej</a>!</p>
<p>GitHub Pages is designed to serve one of a specific set of files as a default landing page for your site. The preferred files to be served are index.html, index.md, and README.md, in that order.</p>
<p>README.md is the only such file that exists in your repository’s master branch, and this explains why it is being served when you visit <a href="https://michaelanokyej.github.io/portfolio-manager" rel="nofollow noopener">https://michaelanokyej.github.io/portfolio-manager</a>.</p>
<p>In order to have your React site deployed correctly, you’ll need to ensure that a bundled, production-ready version of it is stored directly in the root of your repository’s publishing source. There are various tools designed to assist in achieving this, one of which “create-react-app”. Documentation on how to deploy to GitHub Pages using “create-react-app” can be found here:</p>
<p><a href="https://create-react-app.dev/docs/deployment#github-pages" rel="nofollow noopener">https://create-react-app.dev/docs/deployment#github-pages</a></p>
<p>I hope that helps!</p>
<p>Friday</p>
